### ISSUE TITLE
Fish autorigging creating expressions error

### ISSUE THREAD

**Issue by MegaVaha** [RxLaboratory/Duik#1201]
# Bug Report

## Environment

- Duik *17.1.11*
- Windows *64 10.0*
- After Effects *23.6x62*

## Report

Hello!
1. I've created fish structure with default OCO metarig Fish of Angela. 
2. Selected all bones (Ctrl+A or selecting box area).
3. Clicked Autorig.
4. Got 4 errors about unexisting of a some parameters for the expressions.

I've attached zip file: aep with already autorigged fish with errors in expressions inside + screen recording with issue.
[duik 17 OCO fish autorigging error aep+screen_rec.zip](https://github.com/RxLaboratory/Duik/files/14001278/duik.17.OCO.fish.autorigging.error.aep%2Bscreen_rec.zip)

**Comment by MegaVaha** [comment]
When I decided to try autorigging of fish, that was builded NOT from OCO characters library fish template BUT from the bones panel of Angela with Fish button (2 step: 1- Fish spine, 2 - fish fin) - I notice, that duiring autorigging of this builded fish there is junping error too: not 4 errors, as in my upper message, but 1 error.

Another my question: what is purpose of the Null, that named "N < Fin >" and was born, when autorigged the fish structure?

In the attached ZIP, please, see aep with error of case of 2-step created fish rigging expressions error and screen rec of process, that finished with error.
[duik 17 2-stap builded fish rigging expressions error demo.zip](https://github.com/RxLaboratory/Duik/files/14001604/duik.17.2-stap.builded.fish.rigging.expressions.error.demo.zip)

**Comment by MegaVaha** [comment]
I created 2-step fish structure (1 - created spine, 2 - fin) and tried to autorig them separately.
1st I selected all body layers of structure and Autorig it. Ok - it was done without errors. 
2nd I selected all fin layers of structure and Autorig it - and I got the same error, that in my previous message (about 1 error when autorig 2-step-created fish structure).

So, the bug is in the Autorig of the Fin structure only.
